content/docs/user-guide/external-dependencies.md

```diff
@@ -41,64 +41,57 @@ stage to your list of stages in dvc.yaml.
 > Note that some of these commands use the `/home/shared` directory, typical in
 > Linux distributions.
 
-### Local file system path
-
-```dvc
-$ dvc run -n download_file
-          -d /home/shared/data.txt \
-          -o data.txt \
-          cp /home/shared/data.txt data.txt
-```
-
-### SSH
-
-```dvc
-$ dvc run -n download_file
-          -d ssh://user@example.com:/home/shared/data.txt \
-          -o data.txt \
-          scp user@example.com:/home/shared/data.txt data.txt
-```
-
 ### Amazon S3
 
 ```dvc
 $ dvc run -n download_file
-          -d s3://mybucket/data.txt \
-          -o data.txt \
-          aws s3 cp s3://mybucket/data.txt data.txt
+          -d s3://mybucket/data \
+          -o data \
+          aws s3 cp s3://mybucket/data data
 ```
 
 ### Microsoft Azure Blob Storage
 
 ```dvc
 $ dvc run -n download_file
-          -d azure://my-container-name/data.txt \
-          -o data.txt \
+          -d azure://my-container-name/data \
+          -o data \
           az storage copy \
                      -d data.json \
                      --source-account-name my-account \
                      --source-container my-container-name \
-                     --source-blob data.txt
+                     --source-blob data
 ```
 
 ### Google Cloud Storage
 
 ```dvc
 $ dvc run -n download_file
-          -d gs://mybucket/data.txt \
-          -o data.txt \
-          gsutil cp gs://mybucket/data.txt data.txt
+          -d gs://mybucket/data \
+          -o data \
+          gsutil cp gs://mybucket/data data
 ```
 
+### SSH
+
+```dvc
+$ dvc run -n download_file
+          -d ssh://user@example.com/path/from/sftp/root/to/data \
```

Comment:
I'm not sure .. but I think it got too complicated

Reply:
I thought so too at first as discussed in https://github.com/iterative/dvc.org/pull/1649#pullrequestreview-459592083 but this way a longer note about paths relative to SFTP root is avoided.

Reply:
I'm happy to go back to a shorter sample path like just `/data` and expand on the note below. But what would we do with the other `path/from/sftp/root` example URLs in get/import-url ? May need to add a similar note in those docs, which we can do.